### PR TITLE
Add hao-nan-li to on vacation list

### PR DESCRIPTION
**Pull request by hao-nan-li**
**Release Note Template for Downstream PRs (will be copied)**

```release-note:none

```

### CODE REVIEW

File: .ci/magician/github/membership.go [hunk 1]
```diff
@@ -22,15 +22,16 @@ var (
 		"roaks3",
 		"zli82016",
 		"trodge",
-		"hao-nan-li",
```
Comment (zli82016) [inline .ci/magician/github/membership.go:25]: You can stay here and only change the `onVacationReviewers` list.

Comment (zli82016) [reply .ci/magician/github/membership.go:25]: This is the reason for build-and-unit-tests failed.

Comment (hao-nan-li) [reply .ci/magician/github/membership.go]: Thanks for catching it!

**Review by zli82016 — APPROVED** [review]
LGTM

**Pull request merged**